src/Utils/Build.php

```diff
@@ -10,14 +10,16 @@
  * address lookup plugin concatenating each property using the specified
  * glue in order to set input value.
  */
-class Build
+final class Build
```

Comment:
why final?

Reply:
the internal call to static gives a stan error, and this class is not intended to be reused so i think is the easy way to avoid it

Reply:
even this has the same stan error and the same future use as Build class : https://github.com/abbadon1334/atk4-google-address/blob/cb472c3c3197e26ce0a724771ed51bcb25183c06/src/Utils/Value.php#L7

final is the best option, those classes are intended for internal usage strictly related to GoogleAPI

Reply:
I prefer non final, final is sometimes very hard to overcome to workaround some problems on user side.